### PR TITLE
MidiKeyboard: add channel spinner and program control

### PR DESCRIPTION
**Pull request by philburk**
Add buttons for incrementing program by +/- 10 or +/- 1. 
Middle buttons shows current program. Also sends program if pressed.

### CODE REVIEW

File: MidiKeyboard/src/com/mobileer/midikeyboard/MainActivity.java [hunk 1]
```diff
@@ -39,7 +44,24 @@
 
     private MidiInputPortSelector mKeyboardReceiverSelector;
     private MusicKeyboardView mKeyboard;
+    private Button mProgramButton;
     private MidiManager mMidiManager;
+    private int mChannel; // ranges from 0 to 15
+    private int[] mPrograms = new int[MidiConstants.MAX_CHANNELS]; // ranges from 0 to 127
+    private byte[] mByteBuffer = new byte[3];
+
+    public class ChannelSpinnerActivity extends Activity implements AdapterView.OnItemSelectedListener {
```
Comment (loki666) [inline MidiKeyboard/src/com/mobileer/midikeyboard/MainActivity.java]: extends Activity ?

Comment (philburk) [inline MidiKeyboard/src/com/mobileer/midikeyboard/MainActivity.java]: Oops! I copied this example, not realizing it was an entire Activity. 
https://developer.android.com/guide/topics/ui/controls/spinner.html

It work if I remove "extends Activity". Thanks for spotting that.

Comment (loki666) [inline MidiKeyboard/src/com/mobileer/midikeyboard/MainActivity.java]: Usually, I don't bother creating a named class for a Listener, I just instantiate a new object in the .setOnItemSelectedListener()

Comment (philburk) [inline MidiKeyboard/src/com/mobileer/midikeyboard/MainActivity.java]: I normally do that as well.

**Pull request merged**